web/src/popover_menus.ts

```diff
@@ -146,6 +146,10 @@ function get_popover_items_for_instance(instance: PopoverInstance): JQuery | und
         return undefined;
     }
 
+    if (is_personal_menu_popover_displayed()) {
+        return $current_elem.find("a:visible, #theme-switcher .tab-option-content");
+    }
```

Comment:
Would it be reasonable to just include `.tab-option-content` in the general selector below, rather than adding a conditional?

I guess the best setup would be to have a more explicit signal that it looks for, but that seems like it'd be OK as a generic tweak, especially with a bit of a comment.

Reply:
Thoughts about using `[tabindex='0']` as the selector? I see that, some `a` tags don't have an explicit `tabindex="0"` on them, so we would probably also have to keep the `a:visible`.

Reply:
Resolved following a dicussion at [CZO](https://chat.zulip.org/#narrow/stream/6-frontend/topic/Use.20.5Btabindex.3D'0'.5D.20selector.20to.20query.20elements.20for.20navigation/near/1761130).